cli/internal/git.go

```diff
@@ -141,14 +143,21 @@ func Clone(repositories ...Project) {
 func GetBranch(gitRepositoryDir string) string {
 	args := []string{"rev-parse", "--abbrev-ref", "HEAD"}
 
-	return Execute(gitRepositoryDir, "git", args[0:]...)
+	branch, err := shell.Execute(gitRepositoryDir, "git", args[0:]...)
+	if err != nil {
+		return ""
```

Comment:
Given this, should we add some checks to ensure that on L55 we don't have an empty string?

Reply:
Yes, this is something I added on purpose to be able to continue with the test side. Now that the feature file passes locally, I can dedicate more time for this

Reply:
I reviewed that part and noticed it's dead code that I've removed in last commit (ba1e5d5f166f2c260bd175268681e07b7aa99cef)